src/views/Editor.vue

```diff
@@ -12,11 +12,11 @@
 				</b-button-group>
 
 				<b-input-group size="sm" class="w-25 mx-1" prepend="Where are my files">
-					<b-form-select value="fairdata" v-model="selectedSchema" placeholder="None" :disabled="!!selectedSchema" @change="selectSchema">
+					<b-form-select value="fairdata" v-model="selectedSchema" :disabled="!!selectedSchema" @change="selectSchema">
 						<optgroup :label="bundle" v-for="(bundle, index) in bundles" :key="index">
 							<option :value="val" v-for="(val, id) in getSchemas(bundle)" :key="id">{{ !selectedSchema ? val.name : val.shortName }}</option>
 						</optgroup>
-						<option v-if="selectedSchema === null" :value="null">None</option>
+						<option v-if="selectedSchema === null" :value="null" hidden>Select one</option>
```

Comment:
does the `hidden` attribute work? I presume that it should hide the option from the form-select but I did not see such API definition.

When looking at https://bootstrap-vue.js.org/docs/components/form-select/ there would be that `disabled` instead. Also if this `disabled` would be the one then we probably want to add the item as first item with something like:
```
<template slot="first">
  <option v-if="selectedSchema === null" :value="null">-- Please select one --</option>
</template>
```

Reply:
Hidden is a global attribute so it can in theory be used on any HTML element. The hidden option is not visible in the dropdown at all (worked at least with Firefox and Chrome). A disabled element is visible in the dropdown but gray and unselectable. I prefer hidden option in this case.

Reply:
I agree, you are correct. For some reason it didn't work the first time when I tested it at https://bootstrap-vue.js.org/docs/components/form-select/